assets/sass/_themes.scss

```diff
@@ -10,8 +10,9 @@ $sidebar-item-icon-width: 1.5rem;
 
 @import 'themes/dark';
 @import 'themes/light';
+@import 'themes/nord';
```

Comment:
There's a missing part  in my [post](https://convos.by/2019/11/2/custom-styling.html) about the order: When it's a custom theme, the theme should be before "themes/light". Please re-order it.